app/src/main/java/org/greenstand/android/TreeTracker/camera/Camera.kt

```diff
@@ -75,6 +75,7 @@ fun Camera(
                         )
                         .build()
 
+                    cameraProvider.unbindAll()
```

Comment:
This seems like it does a reset right before we load the camera. Would it work better if we unbindAll right after taking the picture?

Reply:
I was getting an error that I was trying to bind too many usecases. So it wasn't able to take/save the picture and this fixed it.